bin/npg_move_runfolder

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env perl
+#########
+# Author:        js10
+# Created:       2018-05-30
+#
+
+use strict;
+use warnings;
+
+use FindBin qw($Bin);
+use lib ( -d "$Bin/../lib/perl5" ? "$Bin/../lib/perl5" : "$Bin/../lib" );
+use Carp;
+use autodie qw(:all);
+
+use Monitor::Staging;
+use Monitor::RunFolder::Staging;
+
+our $VERSION = '0';
+
+use Readonly;
+
+my $mon = Monitor::RunFolder::Staging->new_with_options(status_update => 0);
+my @ms = $mon->move_to_analysis();
+print join "\n", @ms;
+
+1;
+
+__END__
+
+
+=head1 NAME
+
+npg_move_runfolder - move 'incoming' to 'analysis' and change group.
+
+=head1 VERSION
+
+=head1 SYNOPSIS
+
+    C<npg_move_runfolder --runfolder_path path>
+
+=head1 DESCRIPTION
+
+This script moves 'incoming' to 'analysis' for a given runfolder,
+and changes the group name to that found in the staging configuration.
+
```

Comment:
The description seems strange. Should read "This script moves a given run folder from 'incoming' directory to 'analysis' directory. "group name to that found in the staging configuration" is rather criptic.